sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1568,6 +1568,16 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val FILESTREAM_SINK_METADATA_IGNORED =
+    buildConf("spark.sql.streaming.fileStreamSink.metadata.ignored")
```

Comment:
Following the [guideline for naming configurations](https://github.com/apache/spark/blob/c6109ba9181520359222fb032d989f266d3221d8/core/src/main/scala/org/apache/spark/internal/config/ConfigEntry.scala#L20-L47), maybe the config can be named like `spark.sql.streaming.fileStreamSink.ignoreMetadata` or `spark.sql.streaming.fileStreamSink.formatCheck.enabled`, or any other good names :)

Reply:
Personally `spark.sql.streaming.fileStreamSink.ignoreMetadata` sounds better. I couldn't get what `formatCheck` means intuitively.

Reply:
`spark.sql.streaming.fileStreamSink.ignoreMetadata` sounds good.